simpledotorg/simple-server
# 3392: scope monthly district data report to currently selected month

**Story card:** [ch3392](https://app.clubhouse.io/simpledotorg/story/3392/add-ability-to-make-reports-starting-on-past-months?ct_workflow=all&vc_group_by=day)

## Because

The recently created monthly district data csv followed the pattern established by the existing csv report downloads, which scope the report to the current month. We would like to be able to create these reports for any month. 

## Test instructions
- Go to the reports tab and navigate to a district
- change the month dropdown to a different month
- click the download tab
- there should be a new option "Monthly district data"
- click that and accept the download
- expect it to be populated with data for the district and its facilities, scoped to the selected month
